lms/djangoapps/grades/migrations/0003_config_model_addition.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.conf import settings
+import xmodule_django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('grades', '0002_rename_last_edited_field'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CourseAuthorization',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('change_date', models.DateTimeField(auto_now_add=True, verbose_name='Change date')),
+                ('enabled', models.BooleanField(default=False, verbose_name='Enabled')),
```

Comment:
Two different things. This model is per-course, and can be explicitly enabled, explicitly disabled, or not present (implicitly disabled, using the current logic).

`SubsectionGradesEnabledFlag` is a global value that is independent of courses.